.husky/pre-commit

```diff
@@ -2,6 +2,7 @@
 . "$(dirname "$0")/_/husky.sh"
 
 # always lint these
+npm run lint:syncpack
```

Comment:
might be assuming too much: if you don't use nix, you need to have shellcheck and actionlint installed globally.

Reply:
`shellcheck` can be installed from NPM and could be added as a dev dependency to the project. Doesn't seem to be the case for `actionlint`.

Reply:
let me clean up a bit of the shell scripts that have these dependencies.

Reply:
Cleaned up with some warning messages.